tests/puppeteer/campaigns/functional/BO/customers/customers/02_CRUDCustomerInBO.js

```diff
@@ -55,6 +55,7 @@ describe('Create, Read, Update and Delete Customer in BO', async () => {
       this.pageObjects.boBasePage.customersParentLink,
       this.pageObjects.boBasePage.customersLink,
     );
+    await this.pageObjects.boBasePage.closeSfToolBar();
```

Comment:
Maybe add the global capability to check if the Symfony toolbar should be closed. Or run test in production env only?

Reply:
We are already trying to execute tests in production env with `PS_DEV_MODE=0` set in .env file for docker, but while testing it, i got a screenshot with the sf toolbar (so i'm confused).
Maybe we are not doing it the right way.

Reply:
It's maybe a problem with the docker image :/

Reply:
For now, we need this to close sf toolbar, and maybe to keep nightly results green (hope So).